backend/src/main/java/com/bakdata/conquery/models/query/queryplan/aggregators/specific/DurationSumAggregator.java

```diff
@@ -50,7 +59,10 @@ public DurationSumAggregator doClone(CloneContext ctx) {
 
 	@Override
 	public Long getAggregationResult() {
-		return set.isEmpty() ? null : set.countDays();
+		if (getColumn().equals(validityDateColumn) && queryDateAggregator.isPresent()) {
```

Comment:
denke den ersten check kannst du dir sparen. sieht auch ziemlich hacky aus so

Reply:
getColumn().equals(validityDateColumn) in next Table integrieren

Reply:
entfällt